sig-security-tooling/cve-feed/hack/fetch-cve-feed.sh

```diff
@@ -25,7 +25,7 @@ apt-get install -y python3-pip
 pip3 install requests
 
 #python script to generate official-cve-feed.json 
-python3 fetch-official-cve-feed.py
+python3 fetch-official-cve-feed.py | tee official_cve_feed.json
```

Comment:
Can you add a note on what `tee` is doing here?

Reply:
Sure